docs/source/first_steps/bashrc.rst

```diff
@@ -7,13 +7,12 @@ Shell environment on Discovery
 The Discovery Shell environment and .bashrc
 +++++++++++++++++++++++++++++++++++++++++++
 
-Discovery uses a Linux-based Operating System (CentOS) that uses the Shell to interface with the user. Bash (Bourne Again SHell) is one of the most popular Shell implementations whichn is the default shell on Discovery.  
+Discovery uses a Linux-based Operating System (CentOS) that uses the Shell to interface with the user. Bash (Bourne Again SHell) is one of the most popular Shell implementations which is the default Shell on Discovery.  
 
-The Shell script .bashrc is used by Bash to initialize your Shell environment. For example, it can be used to define shell environemnt variables (such as `PATH`) or aliases and load modules.
+The Shell script .bashrc is used by Bash to initialize your Shell environment. For example, it can be used to define shell environemnt variables (such as ``PATH``) or aliases and load modules.
```

Comment:
Environment variables should generally go in `~/.bash_profile` or `~/.profile`.  `~/.bashrc` is primarily for aliases and functions.